docs/content/documentation/deployment/vercel.md

```diff
@@ -23,3 +23,76 @@ You can learn more about how to setup a custom domain and how to get the most ou
 [via their documentation.](https://vercel.com/docs) 
 
 After you click the blue "Deploy" button, it's off to the races!
+
+## Troubleshooting
+
+### `GLIBC_X.XX` not found
+
+This is because Vercel's build images comes with an older glibc version whereas Zola 
+depends on a newer glibc. However, Vercel provides a newer build image which can be used in
+deployments by setting Node.js version to "20.x", allowing Zola to work properly.
+
+## Additional options
+
+### Enable trailing slashes
+
+Visiting a page without trailing slash may break relative paths, so you might want to configure
+Vercel to always redirect paths with a trailing slash. By default, redirecting to a trailing
+slash is not enabled on Vercel.
+
+If enabled, for example the `/about` path will redirect to `/about/`. Paths with a file extension 
+will not redirect to a trailing slash, for example `/styles.css` will stay as-is.
+
+To do that, create a file in the root of your git repository named `vercel.json`
+(if it doesn't exists already), and set this option:
+
+```json
+{
+    "trailingSlash": true
+}
+```
+
+### Prefer clean URLs
+
+When enabled, all HTML files will be served without their file extension, so visitors will see
+`/about` instead of `/about.html` while navigating through your website.
```

Comment:
There won't be a `about.html` file with zola, how does that work?

Reply:
My bad, it is meant to be an example to tell how it works, I forgot to indicate that it is an example.

Reply:
I edited the sentence, hope it's better now.